src/pipelines/epidemiology/ro_covid19romania.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from typing import Any, Dict, List
+from pandas import DataFrame, concat, merge
+from lib.io import read_file
+from lib.pipeline import DataSource
+from lib.time import datetime_isoformat
+from lib.utils import table_rename, table_multimerge
+
+
+class Covid19RomaniaDataSource(DataSource):
+    def parse(self, sources: List[str], aux: Dict[str, DataFrame], **parse_opts) -> DataFrame:
+
+        data_list = []
+        for idx, var in enumerate(["total_recovered", "current_intensive_care"]):
+            df = DataFrame.from_records(json.load(open(sources[idx]))["values"])
+            data_list.append(table_rename(df, {"value": var}))
+
+        data = table_multimerge(data_list)
+        data["key"] = "RO"
```

Comment:
Food for thought: would it be worth defining a constant map for country codes?

```python
COUNTRY_ISO_CODES: {
  "UNITED_STATES": "US",
  # ...
  "CHINA": "CN",
  # etc....
}
```

Reply:
We have that in CSV form: https://github.com/open-covid-19/data/blob/master/src/data/metadata.csv